frontend/src/api/client.ts

```diff
@@ -24,6 +24,7 @@ const ALLOWED_UNAUTHENTICATED_PATHS = [
     'widget',
     '/product/',
     'check-in',
+    'csrf-cookie',
```

Comment:
This array relates to frontend path. From what I can see `csrf-cookie` is a backend path.